packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -378,6 +386,7 @@ export default function UpgradeModal({
     </p>
   );
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I'll remove `trialAndUpgradeTreatment` in a follow up PR where I remove all trial flows